core/module/ancestors_spec.rb

```diff
@@ -2,14 +2,21 @@
 require File.expand_path('../fixtures/classes', __FILE__)
 
 describe "Module#ancestors" do
-  it "returns a list of modules included in self (including self)" do
-    BasicObject.ancestors.should == [BasicObject]
-    ModuleSpecs.ancestors.should         include(ModuleSpecs)
-    ModuleSpecs::Basic.ancestors.should  include(ModuleSpecs::Basic)
-    ModuleSpecs::Super.ancestors.should  include(ModuleSpecs::Super, ModuleSpecs::Basic)
-    ModuleSpecs::Parent.ancestors.should include(ModuleSpecs::Parent, Object, Kernel)
-    ModuleSpecs::Child.ancestors.should  include(ModuleSpecs::Child, ModuleSpecs::Super, ModuleSpecs::Basic, ModuleSpecs::Parent, Object, Kernel)
+  def without_test_classes(klasses)
+    ignore = %w{MSpec PP::ObjectMixin ModuleSpecs::IncludedInObject MainSpecs::Module ConstantSpecs::ModuleA}
+    klasses.reject {|k| ignore.any? {|i| k.name.start_with?(i) } }
   end
```

Comment:
This method should be in the fixtures to make sure it's defined on a known module (currently it would be in some Object.new's singleton class, one object per file).